carbon-design-system/ibm-security
# fix: modular styles for tag, tagwall, panel, tearsheet

## Affected issues

- Related to #44

## Proposed changes

Modular styles for
- Panel
- Portal
- Interactive tag
- Tag
- Tag wall
- Tag wall filter
- Tearsheet
- Tearsheet small

## Testing instructions

Feel free to pull this repo down and add import './_index.scss'; to each respective story in this PR...
All components behave as expected except for TimePicker and Dropdown. More investigation needed here but I am thinking this is an issue with carbon-components and not on our end
